pom.xml

```diff
@@ -857,8 +857,8 @@
     <jaxb-api.version>2.3.1</jaxb-api.version>
     <servlet.api.version>3.1.0</servlet.api.version>
     <wx.rs.api.version>2.1.1</wx.rs.api.version>
-    <glassfish.jsp.version>2.3.2</glassfish.jsp.version>
-    <glassfish.el.version>3.0.1-b08</glassfish.el.version>
+    <tomcat.jasper.version>8.5.99</tomcat.jasper.version>
```

Comment:
Ah you are right. Tried git ui first time to resolve and messed up. Thanks for noticing. Let me resolve conflict from git terminal.